OpenOversight/app/utils/forms.py

```diff
@@ -101,8 +101,8 @@ def add_officer_profile(form: AddOfficerForm, current_user: User) -> Officer:
             # don't try to create with a blank string
             if description["text_contents"]:
                 new_description = Description(
-                    description=description["text_contents"],
-                    user_id=current_user.get_id(),
+                    text_contents=description["text_contents"],
+                    created_by=current_user.id,
```

Comment:
Could you add last_updated_by as well?

Reply:
Added last_updated_by and created_by to all other objects created by add_officer_profile as well!